examples/convert_empty_room.py

```diff
@@ -0,0 +1,91 @@
+"""
+.. _ex-convert-mne-sample:
+
+======================================
+Storing empty room data in BIDS format
+======================================
+
+This example demonstrates how to store empty room data in BIDS format
+and how to retrieve them.
```

Comment:
A brief sentence about what emptyroom data is would be nice for EEG only people (who would be confused about this)... Like ".. used for MEG ... see this link"

Reply:
Okay improved the narrative